downgrade

```diff
@@ -4,6 +4,30 @@
 export TEXTDOMAIN=downgrade
 export TEXTDOMAINDIR=/usr/share/locale
 
+usage() {
+  cat <<EOF
+Usage: downgrade [options] [packages] -- [pacman_options]
```

Comment:
I'm not 100% expert in usage syntax, but I think this should be:

```suggestion
Usage: downgrade [options] [packages] [-- [pacman_options]]
```

Thank you for considering compatibility, and it's nice that hyphen-prefixed package names still work.

I'm willing to change things and bump the major version if you think the usage with `[--] [packages]` is nicer. We can ship this one as-is with a minor version bump, then switch that later, or do it as part of this. Up to you.

Reply:
Not an expert on syntax either, but yes the original usage was misleading. I found a nice summary for usage syntax here: http://docopt.org/

Will modify the usage and commit again.